### PR TITLE
add Extract host IP section to logs stream docs

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
This PR closes [Issue 3122](https://github.com/elastic/observability-docs/issues/3122).

### CODE REVIEW

File: docs/en/observability/logs-stream.asciidoc [hunk 1]
```diff
@@ -570,3 +570,291 @@ You should see the following results showing only your high-severity logs:
...
+PUT /logs-example-default/_mapping
+{
+  "properties": {
+    "host.ip": {
+      "type": "ip",
+      "ignore_malformed": true
+    }
+  }
+}
+----
```
Comment (mdbirnstiehl) [inline docs/en/observability/logs-stream.asciidoc:860]: Not sure if this is the recommended way of updating the mapping to ignore malformed IPs. It's what I found in the docs, but let me know if there is a preferred way.

File: docs/en/observability/logs-stream.asciidoc [hunk 2]
```diff
@@ -570,3 +570,291 @@ You should see the following results showing only your high-severity logs:
...
+        ...
+      }
+    }
+  ]
+}
+----
+
+[discrete]
+[[logs-stream-host-ip-query]]
+=== Query logs based on `host.ip`
```
Comment (mdbirnstiehl) [inline docs/en/observability/logs-stream.asciidoc:680]: CIDR notation and range queries were the two ways to query that I found most prevalent. Are there any additional ways to query we might add?

File: docs/en/observability/logs-stream.asciidoc [hunk 3]
```diff
@@ -570,3 +570,291 @@ You should see the following results showing only your high-severity logs:
...
+
+[source,JSON]
+----
+{
+  "docs": [
+    {
+      "doc": {
+        "_index": "_index",
+        "_id": "_id",
+        "_version": "-3",
```
Comment (bmorelli25) [inline docs/en/observability/logs-stream.asciidoc:660]:
```suggestion
        ...
```

I don't think any of this is relevant

File: docs/en/observability/logs-stream.asciidoc [hunk 4]
```diff
@@ -570,3 +570,291 @@ You should see the following results showing only your high-severity logs:
...
+    ]
+  }
+}
+----
+
+[discrete]
+[[logs-stream-ip-ignore-malformed]]
+=== Ignore malformed IP addresses
+
+When you're ingesting a large batch of log data, a single malformed IP address can cause the entire batch to fail. You can prevent this by setting `ignore_malformed ` to `true` for the `host.ip` field. Update the `host.ip` field to ignore malformed IPs using the {ref}/indices-put-mapping.html[update mapping API]:
```
Comment (bmorelli25) [inline docs/en/observability/logs-stream.asciidoc:847]:
```suggestion
When you're ingesting a large batch of log data, a single malformed IP address can cause the entire batch to fail. You can prevent this by setting `ignore_malformed` to `true` for the `host.ip` field. Update the `host.ip` field to ignore malformed IPs using the {ref}/indices-put-mapping.html[update mapping API]:
```

This space is messing things up

**Review by bmorelli25 — APPROVED** [review]
A couple suggestions, otherwise LGTM

File: docs/en/observability/logs-stream.asciidoc [hunk 5]
```diff
@@ -570,3 +570,289 @@ You should see the following results showing only your high-severity logs:
...
+        }
+      }
+    ]
+  }
+}
+----
+
+[discrete]
+[[logs-stream-ip-ignore-malformed]]
+=== Ignore malformed IP addresses
```
Comment (ruflin) [inline docs/en/observability/logs-stream.asciidoc:843]: Nice way of introducing `ignore_malformed` 👍

File: docs/en/observability/logs-stream.asciidoc (hunk 1 above)
Comment (ruflin) [reply docs/en/observability/logs-stream.asciidoc:858]: LGTM

**Pull request merged**